model/src/main/java/com/microsoft/java/bs/gradle/model/JavaExtension.java

```diff
@@ -35,4 +35,9 @@ public interface JavaExtension extends LanguageExtension {
    * The list of compiler arguments.
    */
   public List<String> getCompilerArgs();
+
+  /**
+   * The classes directory.
+   */
+  File getClassesDir();
```

Comment:
The `LanguageExtension` already declares the method `File getClassesDir();`. Is this on purpose? Same to ScalaExtension

Reply:
Fixed